front/src/reducers/map/map.spec.ts

```diff
@@ -0,0 +1,160 @@
+import {
+  MapState,
+  mapInitialState,
+  updateViewportAction,
+  updateMapStyle,
+  updateMapTrackSources,
+  Viewport,
+  transformMapRequest,
+  updateMapSearchMarker,
+  updateLineSearchCode,
+  updateShowIGNBDORTHO,
+  updateShowIGNSCAN25,
+  updateShowIGNCadastre,
+  updateShowOSM,
+  updateShowOSMtracksections,
+  updateFeatureInfoClick,
+  updateLayersSettings,
+  updateSignalsSettings,
+  updateTerrain3DExaggeration,
+  updateTransformRequest,
+} from 'reducers/map';
+import { createStoreWithoutMiddleware } from 'Store';
+
+import { describe, expect } from 'vitest';
+
+const createStore = (initialStateExtra?: MapState) =>
+  createStoreWithoutMiddleware({
+    map: initialStateExtra,
+  });
+
+describe('mapReducer', () => {
+  it('should return initial state', () => {
+    const store = createStore();
+    const mapState = store.getState().map;
+    expect(mapState).toEqual(mapInitialState);
+  });
+
+  it(`should handle updateViewportAction`, () => {
+    const store = createStore();
+    const viewport = {
+      latitude: 40,
+      longitude: 50,
+      zoom: 6.2,
+      bearing: 0,
+      pitch: 0,
+      padding: { top: 0, left: 0, bottom: 0, right: 0 },
+      width: 0,
+      height: 0,
+      transformRequest: transformMapRequest,
+    } as Partial<Viewport>;
+    store.dispatch(updateViewportAction(viewport));
+    expect(store.getState().map.viewport).toEqual(viewport);
```

Comment:
Same suggestion for harmonization with previous PR
```suggestion
   const mapState = store.getState().map
    expect(mapState.viewport).toEqual(viewport);
```